session-4/lecture-4.ipynb

```diff
@@ -2056,7 +2056,7 @@
     }
    ],
    "source": [
-    "features = [name for name in names if 'maxpool' in name.split()[-1]]\n",
+    "features = [name for name in names if 'maxpool' in name.split('/')[-1]]\n",
```

Comment:
The `.split('/')` is not necessary but I think you were trying to split the layer name by `/` but you forget to put it.